pkg/services/ngalert/notifier/images.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"errors"
 
-	"github.com/grafana/alerting/alerting/notifier/channels"
+	"github.com/grafana/alerting/images"
```

Comment:
My comment above can be ignore for these packages as they don't have an alertmanager equivalent.